dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/service/ProcessDefinitionServiceTest.java

```diff
@@ -1068,11 +1068,23 @@ public void testUpdateProcessDefinitionV2() {
                 ((ServiceException) exception).getCode());
 
         // success
+        Mockito.when(processDefinitionLogMapper.queryMaxVersionForDefinition(processDefinition.getCode()))
```

Comment:
we should add more test about the version, including
1. update workflow entity only
2. update task only
3. update task dependence only

all three situation should check workflow, workflow task relation correct or not, and all task still in the workflow